classroom-dashboard/app/assets/stylesheets/application.css

```diff
@@ -81,7 +79,10 @@ div.style_alien_navbar img {
     margin-right: auto;
 }
 
-/*.new_student_link{
+/*
+Are we still using these links?
```

Comment:
this code is not utilized within our application, so we should probably delete it.